### PR TITLE
EDU-2185-MAIN-MENU-PT-wwd

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Translated new menu files from **Work with Domains**
- Removed unnecessary descriptions

[EDU-2185]: https://aziontech.atlassian.net/browse/EDU-2185?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: Configurar domínio
+description: Use Azion Domains para habilitar acesso à sua application por um domínio, permitindo tráfego de usuários e monitoramento de métricas.
+meta_tags: domain, edge computing, edge network, custom domain
+namespace: docs_guides_configure_domain
+permalink: /documentacao/produtos/guias/configurar-dominio/
+---
+
+Com a Azion, você pode criar e executar aplicações diretamente no edge. Para permitir o acesso à sua application através do navegador e começar a receber usuários e monitorar métricas, você precisa de um domínio. Com [Domains](/documentacao/produtos/edge-application/domains/), a Azion fornece um endereço de domínio gratuito no formato `xxxxxxxxxx.map.azionedge.net`. Quando os usuários acessam sua application através deste domínio Azion, o node mais próximo dentro da edge network busca e entrega a edge aplication.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx:9]:
```suggestion
Com a Azion, você pode criar e executar aplicações diretamente no edge. Para permitir o acesso à sua aplicação através do navegador e começar a receber usuários e monitorar métricas, você precisa de um domínio. Com [Domains](/documentacao/produtos/edge-application/domains/), a Azion fornece um endereço de domínio gratuito no formato `xxxxxxxxxx.map.azionedge.net`. Quando os usuários acessam sua aplicação através deste domínio Azion, o node mais próximo dentro da edge network busca e entrega a edge application.
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+:::tip
+Não tem uma edge application? [Construa com Edge Application](link_build_edge_app) ou [use os templates disponíveis](link_templates).
+:::
+
+## Via RTM
+
+Quando você cria uma application [usando um template](link_templates) ou [via CLI](link_cli), ela é automaticamente associada a um domínio Azion. Para criar um novo domínio:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx:17]:
```suggestion
Quando você cria uma application [usando um template](link_templates) ou [via CLI](link_cli), ela é automaticamente associada a um domínio Azion. Para criar um novo domínio personalizado:
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+1. [Acesse o RTM](https://manager.azion.com).
+2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Domains**.
+3. Clique no botão **Add Domain**.
+4. Dê um nome ao seu domínio.
+5. Em **Edge Certificate**, selecione o certificado **Azion SAN**.
+6. Em **Edge Application**, selecione sua edge application.
+7. No campo **CNAME**, você pode listar domínios personalizados para vincular à sua application; por exemplo: `seudominio.com`. Veja a [seção de domínios personalizados](#custom-domains) para saber mais.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx:25]:
```suggestion
7. No campo **CNAME**, você pode listar domínios personalizados para vincular à sua application; por exemplo: `seudominio.com`.
- Veja a [seção de domínios personalizados](#custom-domains) para saber mais.
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+8. Desative o **interruptor de acesso apenas por CNAME**. Isso permitirá que os usuários acessem sua application usando o endereço `xxxxxxxxxx.map.azionedge.net`.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx:26]:
```suggestion
8. Desative o **switch de acesso apenas por CNAME**. Isso permitirá que os usuários acessem sua application usando o endereço `xxxxxxxxxx.map.azionedge.net`.
```

Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx:26]: revisar énfasis e ajustar

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/create-certificate.mdx [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+5. Clique no botão **Save**.
+6. Aguarde alguns minutos para que as alterações se propaguem.
+
+Você pode acessar sua aplicação HTTPS e verificar as configurações de segurança usando seu navegador ou executar o seguinte comando OpenSSL, substituindo `seudominio.com` por um dos CNAMEs do domínio:
+
+```bash
+openssl s_client -showcerts -connect seudominio.com:443
+```
+
+## Via API
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/create-certificate.mdx:41]:
```suggestion

---

## Via API
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx [hunk 6]
```diff
@@ -0,0 +1,250 @@
+---
+title: Migrar nameservers para a Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx:2]:
```suggestion
title: Como migrar nameservers para a Azion
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+permalink: /documentacao/produtos/guias/migrar-ns-para-a-azion
+---
+
+A Azion permite que você hospede e gerencie zonas e registros DNS redirecionando a resolução DNS do seu provedor de domínio para os nameservers (NS) autoritativos da Azion através do [Intelligent DNS](/pt-br/documentacao/produtos/intelligent-dns/).
+
+:::caution[warning]
+Se o **Intelligent DNS** estiver ativado, zonas hospedadas e consultas podem gerar custos relacionados ao uso. Consulte a [página de preços](https://www.azion.com/pt-br/precos/) para mais informações.
+:::
+
+## Criar uma zona DNS
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx:15]:
```suggestion

---

## Criar uma zona DNS
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+            "id": 3211
+        }
+    ]
+}
+```
+
+3. Copie os valores dentro do campo `nameservers`. Esses valores serão usados para [redirecionar o DNS autoritativo do seu domínio](#redirecionar-servidores-dns-autoritativos).
+4. Copie também o valor `id` para quando você [associar uma application à sua zona](#criar-registro-de-edge-application-na-zona-dns).
+
+## Redirecionar servidores DNS autoritativos
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx:89]:
```suggestion

---

## Redirecionar servidores DNS autoritativos
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+
+O próximo passo é redirecionar os servidores DNS autoritativos do seu domínio, e para isso, você precisará acessar seu provedor de domínio e modificar seus registros NS. Aqui estão alguns dos provedores de domínio mais populares e como eles lidam com as mudanças de nameserver DNS:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx:91]:
```suggestion
O próximo passo é redirecionar os servidores DNS autoritativos do seu domínio. Para isso, você precisará acessar seu provedor de domínio e modificar seus registros NS. Aqui estão alguns dos provedores de domínio mais populares e como eles lidam com as mudanças de nameserver DNS:
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx [hunk 10]
```diff
@@ -0,0 +1,250 @@
...
+
+A Azion recomenda que você adicione todos os três nameservers disponíveis da Azion para maior confiabilidade: 
+
+- `ns1.aziondns.net`
+- `ns2.aziondns.com`
+- `ns3.aziondns.org`
+
+Uma vez que seus registros de nameserver tenham sido atualizados para os endereços da Azion, você poderá criar registros diretamente da Azion. Veja a próxima seção para um exemplo de como fazer isso.
+
+## Criar registro de edge application na zona DNS
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx:105]:
```suggestion

---

## Criar registro de edge application na zona DNS
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx [hunk 11]
```diff
@@ -0,0 +1,128 @@
+---
+title: Apontar domínio para a Azion
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx:2]:
```suggestion
title: Como apontar um domínio para a Azion
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx [hunk 12]
```diff
@@ -0,0 +1,250 @@
...
+    "description": "www record",
+    "ttl": 20
+}'
+```
+
+  | Chave | Descrição |
+  | --- | --- |
+  | `record_type` | O tipo de registro sendo criado. Veja a lista completa de tipos de registro aceitos na [documentação do Intelligent DNS](/pt-br/documentacao/produtos/intelligent-dns/). |
+  | `entry` | Define o valor da entrada quando anexado ao domínio apex da zona para gerar o registro. Por exemplo, o valor `www` representará um registro para `www.seudominio.com`. Aceita a notação wildcard (`*`). |
+  | `answers_list` | Aceita uma lista de CNAMEs para os domínios como valores. No exemplo descrito acima, o CNAME é o endereço da Azion. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx:223]: tem outros "endereço Azion"

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+permalink: /documentacao/produtos/guias/apontar-dominio-para-a-azion/
+---
+
+Para direcionar seu tráfego para a Azion, você deve associar as entradas CNAME do seu domínio personalizado ao endereço do seu domínio Azion através de [Domains](/pt-br/documentacao/produtos/edge-application/domains/).
+
+:::note
+Este processo exigirá que você associe um registro CNAME ao seu domínio. Veja [Migrar seu DNS para a Azion](/pt-br/documentacao/produtos/guias/migrar-ns-para-a-azion/) para mais informações sobre como associar um registro ANAME.
+:::
+
+## Verificar domínio personalizado
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx:15]:
```suggestion

---

## Verificar domínio personalizado
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+
+### Via RTM
+
+Para apontar seu domínio Azion para um domínio personalizado, você precisa listar os CNAMEs que deseja configurar para a application:
+
+1. [Acesse o RTM](https://manager.azion.com).
+2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Domain**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx:22]:
```suggestion
2. No canto superior esquerdo da página, abra o **Products menu**, representado por três linhas horizontais, e selecione **Domains**.
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx [hunk 15]
```diff
@@ -0,0 +1,128 @@
...
+}'
+```
+
+
+  | Chave | Descrição |
+  | --- | --- |
+  | `cnames` | Aceita uma lista de CNAMEs para os domínios como valores. Por exemplo, `["*.seudominio.com", "*.seudominio.org"]`. |
+
+:::tip
+Se sua application usa HTTPS, você pode  combinar este processo com a [associação de um certificado digital](/pt-br/documentacao/produtos/guias/criar-certificado-digital/) ao seu domínio personalizado.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx:82]:
```suggestion
Se sua application usa HTTPS, você pode combinar este processo com a [associação de um certificado digital](/pt-br/documentacao/produtos/guias/criar-certificado-digital/) ao seu domínio personalizado.
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+:::
+
+5. Você deve receber uma resposta com os dados atualizados.
+
+## Modificar registros de domínio
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/point-domain-to-azion.mdx:87]:
```suggestion

---

## Modificar registros de domínio
```

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+    "cname_access_only": false,
+    "digital_certificate_id": null,
+    "edge_application_id": <edge_application_id>
+}'
+```
+
+    | Chave | Descrição |
+    | --- | --- |
+    | `name` | Define a string no valor como um nome da entrada do domínio com a Azion. Como este valor não representa um endereço de domínio, você pode modificá-lo para se adequar às suas necessidades. |
+    | `cname_access_only` | Quando definido como `false`, permite o acesso via endereço Azion. |
```
[the remark this answers is not in the document]
Comment (bru-andrade) [reply src/content/docs/pt-br/pages/main-menu/work-with-domains/configure-domains.mdx:51]: To usando os dois, pros propósitos dos guias são a mesma coisa 😬

File: src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx (hunk 12 above)
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/main-menu/work-with-domains/migrate-ns.mdx:223]:
```suggestion
  | `answers_list` | Aceita uma lista de CNAMEs para os domínios como valores. No exemplo descrito acima, o CNAME é o endereço Azion. |
```

**Pull request closed without merging**